rotkehlchen/globaldb/handler.py

```diff
@@ -2103,3 +2109,12 @@ def delete_location_asset_mappings(entries: list[LocationAssetMappingDeleteEntry
                         f'Failed to delete the location asset mapping of {entry.location_symbol} in '  # noqa: E501
                         f'{entry.location} because it does not exist in the DB.',
                     )
+
+    @staticmethod
+    def get_unsupported_assets(location: 'Location') -> set[str]:
+        """Returns the set of unsupported assets in the given location"""
+        with GlobalDBHandler().conn.read_ctx() as cursor:
+            return {asset for (asset,) in cursor.execute(
+                'SELECT exchange_symbol FROM location_unsupported_assets WHERE location = ?',
+                (location.serialize_for_db(),),
+            ).fetchall()}
```

Comment:
I believe you are iterating twice here. Remove the fetchall

```suggestion
            )}
```